contract-address/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "contract-address"
 version = "0.4.0"
 authors = ["Parity Technologies <admin@parity.io>"]
-license = "MIT/Apache2.0"
+license = "MIT/Apache-2.0"
```

Comment:
Maybe switch to [SPDX](https://spdx.org/ids) 2.1 license identifiers like in [serde](https://github.com/serde-rs/serde/commit/6dd2b4607fda3145be9e9c832e7da7bd39cae271)?

Reply:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields
the `/` is deprecated btw